actix-http/src/ws/mask.rs

```diff
@@ -4,16 +4,17 @@ use std::ptr::copy_nonoverlapping;
 use std::slice;
 
 // Holds a slice guaranteed to be shorter than 8 bytes
-struct ShortSlice<'a>(&'a mut [u8]);
+struct ShortSlice<'a>(&'a mut [u8], ());
```

Comment:
This is just so one can't accidentally construct `ShortSlice(foo)`, right? Haven't seen this accomplished using an extra unit field before, why not

```suggestion
struct ShortSlice<'a> {
    inner: &'a mut [u8],
}
```

Reply:
It's kind of a meaningless change either way. The "fix" was making sure to use `::new` everywhere. Just changing the constructor in any way made sure the compiler flagged any additional offences. I like this structure better so will swap.